crates/primitives/starknet/src/transaction/types.rs

```diff
@@ -269,6 +279,8 @@ impl From<Transaction> for InvokeTransaction {
 )]
 #[cfg_attr(feature = "std", derive(serde::Serialize, serde::Deserialize))]
 pub struct Transaction {
+    /// The type of the transaction.
+    pub _type: TxType,
```

Comment:
May `tx_type` be more suitable as Rust uses `_` for non-used variable.
I might be a bit redondant, I agree.